internal/karma/karma.conf.js

```diff
@@ -1,5 +1,56 @@
 // Karma configuration
 // GENERATED BY Bazel
+const path = require('path');
+const fs = require('fs');
+const tmp = require('tmp');
+
+// When karma is configured to use Chrome it will look for a CHROME_BIN
+// environment variable. This line points Karma to use puppeteer instead.
+// See
+// https://github.com/karma-runner/karma-chrome-launcher/blob/master/README.md#headless-chromium-with-puppeteer
+process.env.CHROME_BIN = require('puppeteer').executablePath();
+
+let files = [
+  TMPL_files
+];
+
+const manifestFile = path.join(process.env.TEST_SRCDIR, 'MANIFEST');
```

Comment:
could you add docs on what the code below does?

Reply:
👍